doc/content/documentation/configuration.md

```diff
@@ -0,0 +1,91 @@
++++
+title = "Configuration"
+weight = 15
++++
+
+Phel comes with some configuration options. They are stored in the `composer.json` file under `extra`.
+
+## Structure
+
+These are all Phel specific configuration options available.
+
+```json
+"extra": {
+    "phel": {
+        "loader": {
+            "hello-world\\": "src/"
+        },
+        "loader-dev": {
+            "hello-world\\": "tests/"
+        }
+    }
+}
+```
+
+Until Phel is officialy released, setting `minimum-stability` to `dev` in `composer.json` is required.
+
+```json
+"minimum-stability": "dev"
+```
+
+## Options
+
+This chapter contains all configuration options explained in detail.
+
+
+### `loader`
+
+Autoload mapping for a Phel autoloader.
+
+Under the `loader` key you define a mapping from namespaces to paths, relative to the package root. When autoloading a module like `hello-world\boot` a namespace prefix `hello-world` pointing to a directory `src/` means that the autoloader will look for a file named `src/boot.phel` and include it if present.
+
+Namespace prefixes must end in `\\` to avoid conflicts between similar prefixes. For example `hello` would match modules in the `hello-world` namespace so the trailing backslashes solve the problem: `hello\\` and `hello-world\\` are distinct.
+
+The `loader` references are all added, during install/update, to the Phel runtime which may be found in the generated file `vendor/PhelRuntime.php`.
+
+
+### `loader-dev`
```

Comment:
The explanation is this section is correct. But you need to add that the configuration is identical to `loader`.

Reply:
I don't know what you mean. Can you explain a bit?

Reply:
Something like:

> The `loader-dev` configuration section is equivalent to the `loader` configuration section. Namespaces and paths are defined in the same way.

Reply:
Going to use exactly that :)